spec/support/saml_response_doc.rb

```diff
@@ -185,14 +185,14 @@ def digest_method_nodeset
   end
 
   def transforms_nodeset
-    @_transforms ||= response_doc.xpath(
+    @transforms_nodeset ||= response_doc.xpath(
       '//ds:Reference/ds:Transforms',
       ds: Saml::XML::Namespaces::SIGNATURE,
     )
   end
 
   def transform(algorithm)
-    @_transform ||= transforms_nodeset[0].xpath(
+    transforms_nodeset[0].xpath(
       "//ds:Transform[@Algorithm='#{algorithm}']",
```

Comment:
Since there's an `algorithm` argument that changes the result, I decided it was safer not to memoize this one at all